src/main/java/pl/javastart/foodieapp/item/ItemController.java

```diff
@@ -10,16 +10,16 @@
 @Controller
 public class ItemController {
 
-    private ItemRepository itemRepository;
+    private final ItemService itemService;
 
-    public ItemController(ItemRepository itemRepository) {
-        this.itemRepository = itemRepository;
+    public ItemController(ItemService itemService) {
+        this.itemService = itemService;
     }
 
     @GetMapping("/danie/{name}")
```

Comment:
jak masz jeden@GetMapping nie ma potrzeby dodawania jeszcze w path tego /danie. Wystarczy jak zrobisz w postaci @GetMapping("/{name}). Po drugie to name powinno być jako @RequestParam nie jako @PathVariable (tak zgodnie z zasadami REST API)